easy_phi/app.py

```diff
@@ -124,11 +126,11 @@ def get(self):
 
 # URL schemas to RequestHandler classes mapping
 APPLICATION = tornado.web.Application([
-    (r"/", VersionHandler),  # Default page
-    (r"/version", VersionHandler),
-    (r"/modules", ModulesListHandler),
-    (r"/pick_module", SelectModuleHandler),
-    (r"/send_command", SCPICommandHandler),
+    (r"/", PlatformInfoHandler),  # Default page
```

Comment:
I don't agree. Pick module and Display list of modules are two semantically different operations. First assumes modifying the internal state of the program (locking the module to a particular user), therefore should be implemented using POST request. The second one is a simple query of the internal state of the program, therefore should be implemented using GET request. Since the two operations have nothing in common I suggest to implement them using two separate RequestHandlers